dp/drivers/i40e.c

```diff
@@ -354,9 +354,30 @@ static void promiscuous_disable(struct ix_rte_eth_dev *dev)
 	rte_eth_promiscuous_disable(dev->port);
 }
 
+static DEFINE_SPINLOCK(i40e_dev_lock);
+
 static int reta_update(struct ix_rte_eth_dev *dev, struct rte_eth_rss_reta *reta_conf)
 {
-	panic("reta update not implemented\n");
+	int ret, i;
+	struct rte_eth_rss_reta_entry64 r_reta_conf[2];
```

Comment:
I supposed this `2` is a magic number that should be changed to `ETH_RSS_RETA_NUM_ENTRIES / 64`